crates/viewer/re_viewport/src/viewport_ui.rs

```diff
@@ -277,14 +277,27 @@ impl ViewportUi {
         if ctx.egui_ctx.input(|i| i.pointer.any_released()) {
             egui::DragAndDrop::clear_payload(ctx.egui_ctx);
 
-            for entity in entities {
-                if can_entity_be_added(entity) {
-                    view_blueprint.contents.raw_add_entity_inclusion(
-                        ctx,
-                        EntityPathRule::including_subtree(entity.clone()),
-                    );
-                }
-            }
+            view_blueprint
+                .contents
+                .mutate_entity_path_filter(ctx, |filter| {
+                    for entity in entities {
+                        if can_entity_be_added(entity) {
+                            filter.add_rule(
+                                RuleEffect::Include,
+                                EntityPathRule::including_subtree(entity.clone()),
+                            );
+                        }
+                    }
+                });
+
+            // for entity in entities {
+            //     if can_entity_be_added(entity) {
+            //         view_blueprint.contents.raw_add_entity_inclusion(
+            //             ctx,
+            //             EntityPathRule::including_subtree(entity.clone()),
+            //         );
+            //     }
+            // }
```

Comment:
with this pr I have the context why this is still here but no point in letting it rot. the upcoming change will be the `mutate` function and then things change here again anyways

```suggestion
```

Reply:
I just noticed it a removed it already :)